tests/js/common/aql/aql-view-arangosearch-ddl-rocksdb.js

```diff
@@ -0,0 +1,78 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertTrue, assertFalse, assertEqual */
+
+const jsunity = require('jsunity');
+const tasks = require('@arangodb/tasks');
+
+function testSuite() {
+  const db = require("@arangodb").db;
+  const dbName = 'TestDB';
+  return {
+    setUp: function() {
+      db._useDatabase("_system");
+      try { db._dropDatabase(dbName); } catch (e) {}
+      db._createDatabase(dbName);
+    },
+    tearDown: function() {
+      db._useDatabase("_system");
+      db._dropDatabase(dbName);
+    },
+    ////////////////////////////////////////////////////////////////////////////
+    /// @brief test creating link with 'inBackground' set to true
+    ////////////////////////////////////////////////////////////////////////////
+    testCreateLinkInBackgroundMode: function () {
+      const colName = 'TestCollection';
+      const viewName = 'TestView';
+      const initialCount = 500;
+      const inTransCount = 1000;
+
+      db._useDatabase(dbName);
+      let col = db._create(colName);
+      let v = db._createView(viewName, 'arangosearch', {});
+      // some initial documents
+      for (let i = 0; i < initialCount; ++i) {
+        col.insert({ myField: 'test' + i }); 
+      } 
+      let commandText = function (params) { 
+          var db = require('internal').db; 
+          db._executeTransaction({
+              collections:  { write: params.colName },
+              action: function(params) {
+                var db = require('internal').db; 
+                var c = db._collection(params.colName); 
+                for (var i = 0; i < params.inTransCount; ++i) { 
+                  c.insert({ myField: 'background' + i }); 
+                } 
+                require('internal').sleep(20); 
+              },
+              params: params
+            });
+          };
+      tasks.register({
+        command: commandText,
+        params: { colName, inTransCount, dbName },
+        name: "CreateLinkInBackgroundMode_AuxTask"
+      });
+
+      require('internal').sleep(5); // give transaction some time to run before us
```

Comment:
The 5 seconds here will not guarantee that the task with the transaction has already run or is completed when the sleep is over. If you want to ensure this (and the test depends on it) it should be solved in a different way.